src/os/bluestore/ceph-aio.h

```diff
@@ -11,7 +19,16 @@
 #include "include/types.h"
 
 struct aio_t {
+#if defined(HAVE_LIBAIO)
   struct iocb iocb{};  // must be first element; see shenanigans in aio_queue_t
+#elif defined(HAVE_POSIXAIO)
+  //  static long aio_listio_max = -1;
```

Comment:
what is this comment for? can we remove it?

Reply:
@tchaikov 
Copied it from Alan's woork, and I think it is working up to keeping track of the max items that can go into io_submit. And that needs to be read from sysctl, but not implemented yet.

Reply:
@wjwithagen could you please file a tracker or add a `// TODO` instead? i can hardly deduce the intention of the author by reading this comment.